bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -385,14 +385,8 @@ impl Script {
     /// [BIP143]: <https://github.com/bitcoin/bips/blob/99701f68a88ce33b2d0838eb84e115cef505b4c2/bip-0143.mediawiki>
     pub fn p2wpkh_script_code(&self) -> Option<ScriptBuf> {
         self.p2wpkh().map(|wpkh| {
```

Comment:
FWIW I think the `self.p2wpkh` function is really confusingly named. If it's returning just the hash it should be named `wpkh`. As named, it looks like it's returning the whole script.

But it's only `pub(crate)` so it's fine.

Reply:
I nearly inlined it while doing this, since you mention it I'll take another look.

Reply:
https://github.com/rust-bitcoin/rust-bitcoin/pull/2457